packages/cow-hooks-ui/src/TokenAmount.tsx

```diff
@@ -15,7 +15,9 @@ export function TokenAmount({
   return (
     <span className="text-xs">
       {formatNumber(balance, 4)} {token.symbol}{" "}
-      <i>(≈ ${fiatValue >= 0 ? formatNumber(fiatValue, 2) : "0"})</i>
+      <i className="xsm:block hidden">
```

Comment:
👀 does `xsm` exist? or was this meant to be `xs` or `sm`?

Reply:
Yes, it is one custom width that jean created!